### PR TITLE
Add : scheduler_tracking config

### LINKED ISSUE #647: lr_scheduler config issue
https://github.com/ashleve/lightning-hydra-template/blob/bddbc24b82ab6ccfa6243e815a49dc5bfe8d4144/src/models/mnist_module.py#L202-L213

**There is an issue with not being able to change the monitor interval frequency value of the lr_scheduler using the config.**

If I need to change the learning rate at each step, I will have to modify the module file directly.

### PR DESCRIPTION
**Pull request by GeonHyeock**
## What does this PR do?

**Going forward, the monitor, interval, frequency value of the lr_scheduler can be changed using the config.**

Fixes #647 

## Before submitting

- [x] Did you make sure **title is self-explanatory** and **the description concisely explains the PR**?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you list all the **breaking changes** introduced by this pull request?
- [x] Did you **test your PR locally** with `pytest` command?
- [x] Did you **run pre-commit hooks** with `pre-commit run -a` command?

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by GeonHyeock** [comment]
**Fix errors reported in Git Actions for code-quality and test workflows #643**

The same issue as #643 is occurring.
